website/templates/base.html

```diff
@@ -10,6 +10,7 @@
     <link rel="icon" type="image/png" href="{% static 'img/favicon.png' %}" />
     <link rel="shortcut icon" href="{% static 'img/favicon.ico' %}" />
     <link rel="stylesheet" href="{% static 'css/site.min.css' %}" media="screen">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.2.0/css/all.css" integrity="sha384-hWVjflwFxL6sNzntih27bfxkr27PmbbK/iSvJ+a4+0owXq79v+lsFkW54bOGbiDQ" crossorigin="anonymous">
```

Comment:
You can leave this until the end of this pull request, but since the map is already slow, we shouldn't introduce the need for more external CSS files here. Can we use some basic icons that we already have, or just use the ones that were being used previously?

Reply:
sure